AppCenterDataStorage/AppCenterDataStorage/MSPaginatedDocuments.m

```diff
@@ -2,25 +2,81 @@
 // Licensed under the MIT License.
 
 #import "MSPaginatedDocuments.h"
```

Comment:
sort alphabetically

Reply:
Seems like we have a convention for the h file for the current m file to be at the top always.